src/Virtual_account.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Razorpay\Api;
+
+class Virtual_account extends Entity
```

Comment:
CamelCase. Filename too.

Reply:
The route namespace for this entity is `/virtual_accounts`. I won't be able to use the parent functions defined in the entity class then (Eg: `parent::create($attributes);`). It lowercases the classname and adds an `s` at the end.

Reply:
Then change that logic no? Don't misname your new classes because existing logic is not suiting it. Update the URL generation logic to handle multi-word entities, or else override the url getter functions.